USAGE.md

```diff
@@ -391,6 +392,27 @@ sso_account_id=123456789012
 sso_role_name=Administrator
 ```
 
+## Assuming roles with web identities
+
+AWS supports assuming roles using [web identity federation and OpenID Connect](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-role.html#cli-configure-role-oidc), including login using Amazon, Google, Facebook or any other OpenID Connect server. The configuration options are as follows:
+* `web_identity_token_file` A file that contains an OpenID Connect identity token. The token is loaded and passed as the `WebIdentityToken` argument of the `AssumeRoleWithWebIdentity` operation.
+* `web_identity_token_process` A command that executes to generate an OpenID Connect identity token. The token written to the command's standard out is passed as the `WebIdentityToken` argument of the `AssumeRoleWithWebIdentity` operation.
```

Comment:
I can't find any reference to `web_identity_token_process` in the AWS CLI docs. Is this a new config parameter?

Reply:
Ah yes I see from the PR description that it is.

Although we have endeavoured to retain compatibility with the aws cli config, this seems pretty reasonable.

We should document this further perhaps in the config section of this doc, pointing out it is not standard

Reply:
> Although we have endeavoured to retain compatibility with the aws cli config, this seems pretty reasonable.

Awesome. It's designed to be really similar to the officially-supported `credential_process` option, but for retrieving a web identity token instead of an AWS access key/secret key/session key directly.

> We should document this further perhaps in the config section of this doc, pointing out it is not standard

:+1: What do you think of 4d712c1?